mne/time_frequency/tfr.py

```diff
@@ -1065,7 +1065,7 @@ def save(self, fname, overwrite=False):
 def _prepare_write_tfr(tfr, condition):
     """Aux function"""
     return (condition, dict(times=tfr.times, freqs=tfr.freqs,
-                            data=tfr.data, info=tfr.info, nave=tfr.nave,
+                            data=tfr.data, info=tfr.info._store, nave=tfr.nave,
```

Comment:
IIUC the `_store` is the dict that actually holds the entries. So anyone who has code that relies on `Info` being a `dict` will fail, but our IO can be fixed -- it uses private functions for read/write, so we can write the `_store` and recreate the `Info` object from it on read as well. Or were you thinking of something else?